### ISSUE TITLE
Build Static HTML Templates and Styles

### ISSUE THREAD

**Issue by dansackett** [codezeus/calliope#22]
While I am working on the API and other backend tasks, we need to start building the HTML and CSS for a basic blog design. My thinking is, instead of getting into the React JSX markup, we should build the HTML pages static and style them. This way, once the structure is defined and ready, all I have to do is turn the HTML into JSX and break it into React components with the proper state. 

@chrisburnell how does that sound?

I can setup routes to work on static pages if we need to.

I'm thinking the pages we will need will be something like:

- Home page (List of posts)
- Single post view
- Edit / Create post view
- Review post (not sure how it will work yet but will be a way to add feedback for other authors)
- Backend dashboard
- Login (Logout isn't needed since it will just be a button)
- List authors
- Invite author (In the backend for admins - should be a form that creates a new user with an email, first name, last name, permissions, etc)
- Edit Author

There may be others but off the top of my head, those will be important to have.

**Comment by chrisburnell** [comment]
@dansackett Sounds great man. I'll amend the list if I can think of anything else that needs adding. Any preference for a starting point?

**Comment by dansackett** [comment]
@chrisburnell I'd start with the posts. I'm planning on doing the authentication last still so if we get the post related things done then we'll have 90% of things finished.

**Comment by dansackett** [comment]
This is a nice reference by a member of my local community that might give some insight on sane styling within React. It references the 7-1 pattern that we use so it fits well: http://hugogiraudel.com/2015/06/18/styling-react-components-in-sass/

**Comment by chrisburnell** [comment]
Hugo Giraudel is actually the guy from whom I adopted the 7-1 pattern! I’m 100% on board with this.

**Comment by dansackett** [comment]
Yeah I noticed that after reading the article. This is a nice guest post on his blog.
